sandbox/extension/Humboldt_2023-09-01.xml

```diff
@@ -0,0 +1,87 @@
+    <property group='Organismal Scope' name='excludedLifeStageScope' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/excludedLifeStageScope' dc:relation='https://tdwg.github.io/eco/terms/#eco:excludedLifeStageScope' dc:description='The age classes or life stages of the dwc:Organisms explicitly excluded from sampling during the dwc:Event.' comments='This term is defined based on dwc:lifeStage (http://rs.tdwg.org/dwc/terms/lifeStage). Recommended best practice is to use the same controlled vocabulary as for dwc:lifeStage and to separate multiple values in a list with space vertical bar space ( | ).' examples='`seedling`; `nestling | fledgling`' required='false'/>
+    <property group='Organismal Scope' name='isLifeStageScopeFullyReported' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/isLifeStageScopeFullyReported' dc:relation='https://tdwg.github.io/eco/terms/#eco:isLifeStageScopeFullyReported' dc:description='Every dwc:Organism that was included within the life stage scope, and was detected during the dwc:Event, was reported.' comments='This term is only relevant if the dwc:Event used restricted search or open search methods. If all dwc:Organisms included within the life stage scope and detected during the dwc:Event were reported, the value should be &#x27;true&#x27;. Life stage scope is based on the combination of eco:targetLifeStageScope and eco:excludedLifeStageScope.' examples='`true`; `false`' required='false'/>
+    <property group='Organismal Scope' name='targetDegreeOfEstablishmentScope' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/targetDegreeOfEstablishmentScope' dc:relation='https://tdwg.github.io/eco/terms/#eco:targetDegreeOfEstablishmentScope' dc:description='The degrees of establishment of the dwc:Organisms targeted for sampling during the dwc:Event.' comments='Recommended best practice is to use controlled value strings from the controlled vocabulary (http://rs.tdwg.org/dwcdoe/) for dwc:degreeOfEstablishment. For details, refer to https://doi.org/10.3897/biss.3.38084. Recommended best practice is to separate multiple values in a list with space vertical bar space ( | ).' examples='`native (category A)`; `invasive (category D2) | widespread invasive (category E)`' required='false'/>
```

Comment:
Is the vocabulary (thesaurus) omitted here because it is multi-value?

See https://github.com/gbif/rs.gbif.org/blob/master/core/dwc_occurrence_2022-02-02.xml#L50C1-L50C1 for how DwC uses it

Reply:
Yes, the list makes the controlled vocabulary problematic. The Occurrence Core doesn't have that issue.

Reply:
OK. We can add it if/when the IPT supports multi-selection in thesaurus.